eventhub_client/src/eventhubreceiver_ll.c

```diff
@@ -744,6 +744,8 @@ static int EventHubAMQP_InitializeStackCommon
             hostName = EventHubConnectionParams_GetHostName(connectionParams);
             tlsIOConfig.hostname = hostName;
             tlsIOConfig.port = TLS_PORT;
+            tlsIOConfig.underlying_io_interface = NULL;
```

Comment:
Actually no change is needed in eventhubclient_ll.c, since the field get initialized with 0's there due to the way the code is written.

---
In reply to: [109302384](https://github.com/Azure/azure-event-hubs-c/pull/11#discussion_r109302384) [](ancestors = 109302384)